gromacs/core.py

```diff
@@ -495,7 +495,7 @@ def __init__(self, *args, **kwargs):
         self.failuremode = kwargs.pop('failure','raise')
         self.extra_doc = kwargs.pop('doc',None)
         self.gmxargs = self._combineargs(*args, **kwargs)
-        self.__doc__ = self.gmxdoc
```

Comment:
It was this line that extracted the docstring right in the initialization. As a lot of commands classes are created when we import the library, it slows the tool loading.